moduleframework/avocado_testers/avocado_test.py

```diff
@@ -263,6 +263,19 @@ def run_script(self, *args, **kwargs):
         """
         return self.backend.run_script(*args, **kwargs)
 
+    def mark_as_warn(self, func, *args, **kwargs):
+        """
+        run function which you would like to mark as WARN
+        :param func: function for run
+        :param args: pass this args to run function
+        :param kwargs: pass thru to avocado process.run
```

Comment:
This is not true, this is passed same as args as key-value to `func`, not to avocado process run.